azavea/temperate
# Add support clause to activation email and message

## Overview

Advises the user to reach out to our support email if they are having trouble activating their account.

### Demo

![image](https://user-images.githubusercontent.com/1042475/38050517-4d976866-3299-11e8-9625-a524233eea3d.png)

![image](https://user-images.githubusercontent.com/1042475/38050552-669d6f2c-3299-11e8-928b-d2a43004d4d3.png)

## Testing Instructions

- View the email templates and message text, and verify that it satisfies the change detailed in #844.

Closes #844
